docs/functions.rst

```diff
@@ -98,12 +100,8 @@ Example:
     name: examplePhase
     args:
       namespace: "{{ .Deployment.Namespace }}"
-      pods:
-        - "{{ index .Deployment.Pods 0 }}"
-        - "{{ index .Deployment.Pods 1 }}"
-      containers:
-        - kanister-sidecar1
-        - kanister-sidecar2
+      pods: "{{ index .Deployment.Pods 0 }} {{ index .Deployment.Pods 1 }}"
+      containers: "container1 container2"
```

Comment:
In the args we have specified pods and containers type is `[]string`. Do we need to update that considering it expects comma-separated items in a single string?
```
   `pods`, Yes, `[]string`, list of names of pods in which to execute
   `containers`, Yes, `[]string`, list of names of the containers in which to execute
```

Reply:
looks like it has been address with the latest commit